target/linux/ath79/dts/ar9344_devolo_magic-2-wifi.dts

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,magic-2-wifi", "qca,ar9344";
+
+	aliases {
+		led-boot = &plcr;
+		led-failsafe = &plcr;
+		led-running = &plcw;
+		led-upgrade = &plcr;
```

Comment:
I seem to have overlooked this earlier: At the moment, you are using plcw both for netdev and as status led.
This is discouraged, as you never know whether it's blinking because of ethernet traffic or because of boot/failsafe/whatever.
In the earlier PR, you mentioned a combined led. Is plcr part of this combined led?

Reply:
yes, plcr and plcw are together in the same button (plc button), plcr is plc RED led and plcw is plc white led

Reply:
As mentioned before, I won't accept using an ethernet/Wifi indicator as status LED.

Reply:
Oem behaviour: 
When you plug the device, dlan_red turns on while is booting.
After a while, the device boot and turn off dlan_red. (so all are off now)
After a while, wifi and plc initiate. Now the leds will turn on or not depending on user configuration.

When the device is upgrading, plc_red turns on
 
With this description, this is what I think is right:
	led-boot = &led_plc_red; (oem behaviour) 
	led-failsafe = &led_plc_red; (it is kind of boot stage and it is an special case)
	led-running = &led_plc_white; (we could do it on led_plc_red, but then we MUST NOT left this red led on when we are done, will confuse the user because oem uses red led to signal plc problems, IMHO has more logic use led_plc_white)
	led-upgrade = &led_plc_red; (oem behaviour)

Could you please let me know if you disagree in any of them and what is the value that will be accepted instead?